apps/condo/k6/src/registerBillingReceipt.test.ts

```diff
@@ -0,0 +1,155 @@
+    const organizationId = createdOrganization.json('data.obj.id')
+    const billingIntegration = createBillingIntegration({ token })
+
+    const billingContext = createBillingIntegrationOrganizationContext(
+        { token },
+        { id: organizationId },
+        billingIntegration.json('data.obj'),
+        { status: 'Finished' },
+    )
+    const data = {
+        token,
+        organizationId,
+        billingContext: billingContext.json('data.obj'),
+    }
+    return data
+}
+
+const randomNumber = (numDigits) => {
+    const min = 10 ** (numDigits - 1)
+    const max = 10 ** numDigits - 1
+    return faker.datatype.number({ min, max })
+}
+
+const createAddressWithUnit = () => __ENV.BASE_URL === 'http://localhost:3000'
+    ? `${faker.address.cityName()} ${faker.address.streetAddress(true)}`
+    : 'г Нижний Новгород, пр-кт Ленина, д 88 к 78, кв 1'
+
+const generateServicesData = (count = 3, toPay = '') => {
+    const services = []
+
+    for (let i = 0; i < count; i++) {
+        services.push({
+            id: faker.datatype.number().toString(),
+            name: faker.random.alphaNumeric(),
+            toPay: toPay !== '' ? toPay : faker.datatype.number().toString(),
+            toPayDetails: {
+                formula: 'charge + penalty',
+                charge: faker.datatype.number().toString(),
+                penalty: faker.datatype.number().toString(),
+            },
+        })
+    }
+    return services
+}
+
+const createRecipient = (extra = {}) => ({
+    tin: '3703048756',
+    routingNumber: '046577674',
+    ...extra,
+})
+
+const createJSONReceipt = (extra = {}) => {
+    const [month, year] = dayjs().add(-1, 'month').format('MM-YYYY').split('-').map(Number)
+    return Object.fromEntries(Object.entries({
+        importId: faker.datatype.uuid(),
+        address: createAddressWithUnit(),
+        accountNumber: randomNumber(10).toString(),
+        toPay: faker.finance.amount(-100, 5000),
+        month, year,
+        services: generateServicesData(faker.datatype.number({ min: 3, max: 5 })),
+        ...createRecipient(),
+        raw: extra,
+        ...extra,
+    }).filter(([, value]) => !!value))
+}
+
+export function registerBillingReceipt (data) {
+    const receipts = []
+    const bankAccount = faker.random.numeric(12)
+    const unitName = `${faker.datatype.number({ min: 1, max: 10000 })}`
+    const address = 'г Нижний Новгород, пр-кт Ленина, д 88 к 78'
+    const addressWithUnit = address + ', кв ' + unitName
+    const accountNumberPrefix = faker.datatype.uuid()
+
+    let accountIndex = 0
+    let accountCounter = 0
+    for (let i = 0; i < TOTAL_RECEIPT_NUMBER; i++) {
+        receipts.push(createJSONReceipt({ bankAccount, address: addressWithUnit, accountNumber: `${accountNumberPrefix}${accountIndex}` }))
+        accountCounter++
+
+        if (accountCounter >= RECEIPT_PER_ACCOUNT_NUMBER) {
+            accountCounter = 0
+            accountIndex++
+        }
+    }
```

Comment:
I think it's better to prepare data in `setup` function

Reply:
I don't think this is required to have such complex set up function.
In other hand - setup function supose